mscore/scoreview.cpp

```diff
@@ -2392,18 +2475,33 @@ void ScoreView::cmd(const char* s)
                   is.setString(strg);                       // update status
                   moveCursor();
                   }
-            }
-      else if (cmd == "text-word-left")
+            }},
+      {{"text-word-left"}, [this] {
             toTextBase(editData.element)->movePosition(editData, QTextCursor::WordLeft);
-      else if (cmd == "text-word-right")
+            }},
+      {{"text-word-right"}, [this] {
             toTextBase(editData.element)->movePosition(editData, QTextCursor::NextWord);
-      else if (cmd == "concert-pitch") {
+            }},
+      {{"concert-pitch"}, [this, cmd] {
             QAction* a = getAction(cmd);
             if (_score->styleB(Sid::concertPitch) != a->isChecked()) {
                   _score->startCmd();
                   _score->cmdConcertPitchChanged(a->isChecked(), true);
                   _score->endCmd();
                   }
+            }},
+      };
+
+      auto c = std::find_if(cmdList.begin(), cmdList.end(), [cmd](ScoreViewCmd cc) {
```

Comment:
I suggest that `cc` be `const& ScoreViewCmd`.

Reply:
I take it you mean `const ScoreViewCmd&`?

Reply:
Yes. Whoops.

Reply:
Or, for consistency with `Score::cmd()`, `const auto&`.